src/API/ApiEditSpace.php

```diff
@@ -74,6 +76,9 @@ public function execute() {
 				$new_space->setSpaceAdministrators( $ns_admins );
 			}
 		}
+		if ( $ns_protected !== null ) {
```

Comment:
So if nw_protected has a value of "Harry" then it is protected as well?

Reply:
No, just mention it in the docs... if it is not explicitly "false", then it becomes true.